test/os/ui/query/basicquery.xml

```diff
@@ -79,7 +79,7 @@
     <And>
       <And>
         <Or>
-          <Intersects description="Some Description" namehint="Some Area" xmlns="http://www.opengis.net/ogc">
+          <Intersects description="Some Description" namehint="Some Area" id="ID" xmlns="http://www.opengis.net/ogc">
```

Comment:
It might be useful to include an escaped character here like `&amp;` to verify in the test that it gets brought back to just an `&`

Reply:
This is a great idea!